bigc/resources/customers.py

```diff
@@ -10,24 +10,31 @@ class BigCommerceCustomersAPI:
     def __init__(self, store_hash: str, access_token: str):
         self._v3_client = BigCommerceV3APIClient(store_hash, access_token)
 
-    def all(self, *, id_in: Iterable[int] = None, include_formfields: bool = False) -> Iterator[dict]:
+    def all(self, *, id_in: Iterable[int] = None, include_formfields: bool = False,
+            include_storecredit: bool = False) -> Iterator[dict]:
         """Return an iterator for all customers"""
         url_parts = urlparse('/customers')
 
         query_dict = {}
         if id_in is not None:
             query_dict['id:in'] = ','.join(map(str, id_in))
+        if include_storecredit:
+            query_dict['include'] = 'storecredit'
         if include_formfields:
             query_dict['include'] = 'formfields'
```

Comment:
If someone sets the request to include store credit and form fields, wouldn't this overwrite the setting for store credit?

Reply:
See https://github.com/MedShift/bigc/blob/ced475b05d9424d7612e6ea4c3c4eb056d018fe1/bigc/resources/products.py#L15-L24